python/extensions/python.bzl

```
@@ -12,7 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"Python toolchain module extensions for use with bzlmod"
+"""Python toolchain module extensions for use with bzlmod.
+
+## Basic usage
+
+The simplest way to configure the toolchain with `rules_python` is as follows.
+
+```starlark
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    is_default = True,
+    python_version = "3.11",
+)
+use_repo(python, "python_3_11")
+```
+
+For more in-depth documentation see the {rule}`python.toolchain`.
```

python.toolchain is a tagclass, so `{rule}` won't xref to it. obj should work, though, or `{tag-class}` I think.

```suggestion
For more in-depth documentation see the {obj}`python.toolchain`.
```